pkg/get/tools.go

```diff
@@ -1823,5 +1823,23 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 		{{.Name}}-{{$osStr}}-latest`,
 		})
 
+	tools = append(tools,
+		Tool{
+			Owner:       "kumahq",
+			Repo:        "kuma",
+			Name:        "kumactl",
+			Version:     "1.3.1",
+			Description: "kumactl is a CLI to interact with Kuma and its data",
+			URLTemplate: `
+			{{$osStr := ""}}
+			{{- if HasPrefix .OS "linux" -}}
+			{{$osStr = "ubuntu"}}
+			{{- else if eq .OS "darwin" -}}
+			{{$osStr = "darwin"}}
+			{{- end -}}
+			https://download.konghq.com/mesh-alpine/{{.Repo}}-{{.Version}}-{{$osStr}}-amd64.tar.gz`,
```

Comment:
So, if you use armv7l as the `--arch` override, you will get the wrong binary.

We need to template the `amd64` portion of this string. Why? So that's empty unless it's set otherwise.

For invalid architectures and operating systems we should cause a 404 in the URL.

Also, do they really not have a Windows binary for their CLI?

Reply:
They do not have a windows binary. :disappointed: 
I'll fix the arch problems asap, thanks for the note! :smile: